gem/lib/rb_sys/cargo_builder/link_flag_converter.rb

```diff
@@ -22,14 +22,12 @@ def self.convert_arg(arg)
           ["-L", "native=#{$1}"]
         when /^--library=(\w+\S+)$/, /^-l\s*(\w+\S+)$/
           ["-l", $1]
-        when /^-l\s*:lib(\S+).a$/
-          ["-l", "static=#{$1}"]
         when /^-l\s*:lib(\S+).(so|dylib|dll)$/
           ["-l", "dylib=#{$1}"]
         when /^-F\s*(.*)$/
           ["-l", "framework=#{$1}"]
         else
-          ["-C", "link_arg=#{arg}"]
+          ["-C", "link-arg=#{arg}"]
```

Comment:
Couldn't find any reference to `link_arg` (underscored), I _think_ this should be `link-arg`.

Reply:
oh my nice find 😮